packages/gatsby/src/commands/develop.js

```diff
@@ -92,13 +94,50 @@ async function startServer(program) {
       heartbeat: 10 * 1000,
     })
   )
-  app.use(
+  app.post(
     `/___graphql`,
     graphqlHTTP({
       schema: store.getState().schema,
-      graphiql: true,
+      graphiql: false,
```

Comment:
```suggestion
```

Reply:
Was not sure what the default was, but this is cleaner :)